tests/test_stack.py

```diff
@@ -158,21 +167,10 @@ def test_stack_repr(self):
             "notifications=[sentinel.notification], " \
             "on_failure=sentinel.on_failure, " \
             "stack_timeout=sentinel.stack_timeout, " \
-            "stack_group_config={}" \
+            "stack_group_config={}, " \
+            "launch_action=include" \
             ")"
 
-    def test_repr_can_eval_correctly(self):
-        sceptre = importlib.import_module('sceptre')
-        evaluated_stack = eval(
-            repr(self.stack),
-            {
-                'sceptre': sceptre,
-                'sentinel': sentinel
-            }
-        )
-        assert isinstance(evaluated_stack, Stack)
-        assert evaluated_stack.__eq__(self.stack)
-
```

Comment:
This seemed to be a very strange test. I searched our codebase and nowwhere are we running `eval` on a Stack's repr. It's also a weird expectation that you can run `eval` on a Stack's repr. @zaro0508, @craighurley Do either of you have any context on the history here? I don't think we need to support this functionality. Since my PR has us using an Enum for the `launch_action`, you can't actually do this any more anyway.